app/models/tag.rb

```diff
@@ -40,6 +106,18 @@ def delete_and_update!
         tag_name:       name
       }
     )
-    destroy
+
+    # Create the delete activity.
+    repository.create_activity(
+      :delete,
+      owner:      actor,
+      recipient:  self,
```

Comment:
Here `self` is the instance of `Tag` that has just been deleted. This should break the final render, isn't it?